async.zsh

```diff
@@ -305,6 +323,18 @@ _async_notify_trap() {
 	done
 }
 
+
+# This function causes all workers to inherit their parent's cwd
+_async_inherit_parent_cwd() {
```

Comment:
Is your intent here to indicate a private function (via `_`-prefix)? Considering my comment about `chpwd_functions`, we could consider making this a public function instead, say something like `async_update_worker_pwd` where the caller can choose the directory themselves (e.g. `async_update_worker_pwd my_worker $PWD`).

Reply:
Yeah I will make it "public."  Also why not make it more generic if we are going that direction.  We could have `async_update_worker_variable $var $value` which could also allow any other variables to be updated.  I suppose PWD is a special one in that to update it you need to use `cd` rather than just `var=value`.